astroscaler/groups.py

```diff
@@ -163,7 +167,11 @@ def is_cooling_down(self, cooldown):
             event_time = parse(event['createdAt'])
 
             if event_time > cooldown_limit:
-                logger.warning("Cooldown has not elapsed, cannot scale group: %s", self)
+                logger.warning(
+                    "Cooldown has not elapsed (%s seconds remaining), cannot scale group: %s",
+                    (event_time - cooldown_limit).seconds,
```

Comment:
`event_time` is slightly better than `endtime`. I still think it should be renamed to `cooldown_starttime` or `policy_starttime` or `event_starttime`... and use the same var for here and above